---
title: "Analysis Report 2: Your Title Here"
author: "Charlotte Lopp"
date: "November 10, 2017"
output: github_document
bibliography: references.bib
csl: bioinformatics.csl
---

*Overall, a single-spaced page is about 500 words. So if the guidelines say half of a page, think about writing around 250 words. You can use the wordcountaddin in RStudio to track your progress.*

# Introduction

Add about 1 page here. Must cite at least 5 peer reviewed articles.

# Methods

## Sample origin and sequencing

  In 2012, Seo et al. conducted transcriptome analysis of lung adenocarcinoma in order to identify the somatic mutations and transcriptional variations associated with lung cancer [@seo2012transcriptional]. They collected 200 fresh surgical specimens of primary lung adenocarcinoma from Korean patients who underwent major lunch resection and recorded diagnosis, gender, cancer stage, and smoking status for each patient. Among these 200 cancer patients, 54.5% (n = 109) were females and 58.0% (n = 116) were never-smokers. Of these, 87 cancer tissues whose driver mutations were not detected by screening tests were analyzed by transcriptome sequencing combined with whole-exome (n = 76) and transcriptome (n = 77) seuqencing of matched normal lung tissue samples. Transcriptome sequencing was a powerful method to use for detecting driver mutations in cancer, since not only somatic point mutations but also aberrant RNA variants such as fusion genes and alternative splicing can be examined. All these sequencing experiments were done as described previously. They generated 14,038,673,860 paired-end 101-bp-long reads from RNA sequencing of 164 samples (87 cancer and 77 corresponding normal tissues). On average, the RNA sequencing throughputs were 9.77 and 7.38 Gbp for cancer and normal tissues, respectively. In the whole-exome sequencing of normal tissues, they obtained 32.96-fold read depth per tissue for regions targeted by the exome capture platform used in this study.  
  Li et al. [@li2015rna] downloaded Seo et al.'s RNA-seq data from paired normal and tumor tissues from 34 nonsmoking and 34 smoking patients with lung adenocarcinoma (GEO:GSE40419). They aimed to perform a large scale transcriptome analysis and identify the genes with altered expression in lung cancer development. They also looked for differences in gene expression profiles between smoking and nonsmoking patients in order to better define the impact of smoking in lung cancer development and provide insights that might better individualize treatment strategies for the patients.  

## Computational

  After obtaining the RNA-seq data, Li et al. applied a stringent filter on the data to remove the gene tags with sparse count data. They then used edgeR to identify the genes differently expressed between tumor and normal tissues as well as the genes that varied between nonsmoker and smoker patients. The GLM model in edgeR allowed them to identify the genes that varied between nonsmoker and smoker patients with lung adenocarcinmoa. They identified 175 genes whose expressions were statistically different between nonsmoker and smoker group. For our use of the data, biomartr [@drost2017biomartr] was used to implement straightforward functions for bulk retrieval of all genomci data for the selected files present in databases hosted by the National Center for Biotechnology Information (NCBI) and European Bioinformatics Institute (EMBL-EBI). Trimmomatic [@bolger2014trimmomatic] was used to handle the paired-end data as it was specifically developed to be a more flexible and efficient preprocessing tool. Sailfish [@patro2014sailfish], which is a computational methd for quantifying the abundance of previously annotated RNA isoforms from RNA-seq data, was used because it provides quantification estimates much faster than other existing approaches without loss of accuracy. It exemplifies the potential of lightweight algorithms for efficiently processing sequencing reads by facilitating frequent reanalysis of data and reduicng the need to optimize parameters 

# Results

In addition to a minimum of 4-5 figures/tables (and associated captions), you should include sufficient text in this section to describe what your findings were. Remember that in the results section you just describe what you found, but you don't interpret it - that happens in the discussion.

```{r load-libraries, message = FALSE, echo = FALSE}
# Be sure to install these packages before running this script
# They can be installed either with the intall.packages() function
# or with the 'Packages' pane in RStudio

# load general-use packages
library("dplyr")
library("tidyr")
library("knitr")
library("ggplot2")
library("magrittr")

# this package allows for the easy inclusion of literature citations in our Rmd
# more info here: https://github.com/crsh/citr
# and here:
# http://rmarkdown.rstudio.com/authoring_bibliographies_and_citations.html
library("citr")
```

```{r load-data, message = FALSE, echo = FALSE}
# load the dataset from a compressed binary file
# it gets loaded as an object called "final_table"
# this has 3.4 million rows...so you will need to be thoughtful about
# how you analyze the data so that you don't overwhelm your laptop
load("output/final_compiled_counts/joined_count_data.RData")

# changes all columns names to lower case so Travis doesn't complain
names(final_table) %<>% tolower

# test that it loaded correctly before proceeding
stopifnot(exists("final_table"))
```

```{r make-summary-table, echo = FALSE}
# There are many many genes in this dataset, so we can
# subset it down to just a few here to look for interesting patterns
# in the most highly expressed
top_15 <- final_table %>%
  group_by(gender, genename) %>%
  summarize(mean_count = mean(counts_lengthscaledtpm)) %>%
  arrange(desc(mean_count)) %>%
  head(n = 15)

# then we can use the `kable()` function to make a nicely formatted
# markdown table
top_15 %>%
  kable()
```

**Table 1**: The most highly expressed genes in both genders included *SFTPB* and *EEF1A1*.

```{r make-barplot-of-highly-expressed-genes, echo = FALSE}
# this code uses the same data as above, but use it to make a
# barplot - remember geom_col() is just like
# when you use geom_bar(stat = "identity")
top_15 %>%
  ggplot(aes(x = genename,
             y = mean_count,
             fill = gender)) +
    geom_col(position = "dodge")
```

**Figure 1**: Here we show an example figure caption.

```{r make-boxplot-of-highly-expressed-genes, echo = FALSE}
# here we just want to pull out the unique gene names and turn
# them into a vector so we can use it below to make a boxplot
# we use the pull() funtion to get this as a vector, just like
# we did when making histograms several weeks ago
top_genes <- top_15 %>%
  ungroup() %>%
  select(genename) %>%
  unique() %>%
  pull()

# now we need to filter from the full data set again, because
# we don't just want summary data, we want all the data in
# order to make boxplots
final_table %>%
  filter(genename %in% top_genes) %>%
  ggplot(aes(x = genename,
             y = counts_lengthscaledtpm,
             fill = gender)) +
    geom_boxplot() +
    facet_wrap(~cancer_stage) +
    xlab("Gene Name") +
    ylab("Scaled read counts per gene") +
    ggtitle("Read counts per gene by gender and cancer stage") +
    theme_bw() + # simplifies theme
    theme(axis.text.x = # rotates x axis labels vertically
            element_text(angle = 90,
                         hjust = 1))
```

**Figure 2**: Here we show another example figure caption.

# Discussion

Add around 1-2 pages interpreting your results and considering future directions one might take in analyzing these data.

# Sources Cited
